admin_guide/cluster_management.rst

```diff
@@ -226,6 +226,17 @@ Citus places table rows into worker shards based on the hashed value of the rows
 
 However sharing shards can cause resource contention when tenants differ drastically in size. This is a common situation for systems with a large number of tenants -- we have observed that the size of tenant data tend to follow a Zipfian distribution as the number of tenants increases. This means there are a few very large tenants, and many smaller ones. To improve resource allocation and make guarantees of tenant QoS it is worthwhile to move large tenants to dedicated nodes.
 
+The :ref:`citus_stat_statements <citus_stat_statements>` table can help pinpoint resource usage by tenant. Here's how to see which tenant runs most queries, and the time the statements took collectively:
+
+.. code-block:: sql
+
+  select css.partition_key as key, sum(css.calls) as total_queries, sum(pss.total_time) as total_time
```

Comment:
I am having hard time understanding `the time the statements took collectively` part. 

This is the total time to run all multi-tenant queries that this particular tenant participate, but time is for all tenants in the cluster.

Reply:
Oops, I though the "WHERE tenant_id = 12" would be in the query, differentiating it from the same query run against another tenant. But it's probably a shared parameterized query "WHERE tenant_id = ?". Let me think of how to improve this.